src/integrationTest/resources/environment.properties

```diff
@@ -1,3 +1,6 @@
+database.driverClassName=org.testcontainers.jdbc.ContainerDatabaseDriver
+database.url=jdbc:tc:postgresql:9.6://localhost/claimstore
```

Comment:
From my experience I noticed that database name is ignored by `org.testcontainers.jdbc.ContainerDatabaseDriver `and all tests run against database of name `test`.